src/components/views/ProductPage/ProductPage.js

```diff
@@ -64,138 +66,160 @@ const ProductPage = ({
     addToFavourite(id);
   };
 
-  return (
-    <div className={styles.root}>
-      <div className='container'>
-        <div className='row'>
-          <div className='col-5'>
-            <div className={styles.photo}>
-              <img src={image} alt={name} />
-            </div>
-            {typeof images == 'undefined' ? '' : <ProductImages images={images} />}
-          </div>
-          <div className='col-7'>
+  if (error) return <PageNotFound />;
+  else
+    return (
+      <div className={styles.root}>
+        <div className={styles.menu}>
+          <ul>
+            <li className='col-8'>FURNITURE</li>
```

Comment:
To nie poinno być w tej liście, tylko raczej jako nagłówek `<h2>` występować poza listą